DOCSP-18071: fix connection guide (#499)
mongodb/docs-node: source/fundamentals/connection/connect.txt

Connection Guide
================

> .. default-domain:: mongodb

Reviewer: Suggestion:
This directive doesn't do anything anymore, please remove when you see it.

.. figure:: /includes/figures/connection-string-parts.png
   :alt: Each part of the connection string

In this example, we use ``mongodb`` for the protocol, which specifies
the :manual:`Standard Connection String Format </reference/connection-string/#std-label-connections-standard-connection-string-format>`.

.. tip::

   You can also use the :manual:`DNS Seed List Connection Format
   </reference/connection-string/#dns-seed-list-connection-format>` if
   you want more flexibility of deployment and the ability to change the

>    servers in rotation without reconfiguring clients.

Reviewer: Suggestion: 
I think originally we needed to present the DNS Seedlist Connection Format first because that's what Atlas defaults to. Has anything changed such that we're choosing the non seedlist format first?

Reviewer: I see the bullet point on the ticket mentioning moving this to an admonition. This is an old ticket (Aug 2021) that hasn't been kept up to date.
Could you skip this item for this PR? Thanks.

Author: I presented it second because the image of the connection string uri changed and now uses the `mongodb` protocol. But, I will move the info back out of the admontion!

>    You can also use the :manual:`DNS Seed List Connection Format
>    </reference/connection-string/#dns-seed-list-connection-format>` if
>    you want more flexibility of deployment and the ability to change the
>    servers in rotation without reconfiguring clients.

Reviewer 2: I: "also" sort of suggests it's an add-on to the standard format, rather than an alternative

```suggestion
   If your instance or deployment has a DNS SRV record, you can use the 
   :manual:`DNS Seed List Connection Format
   </reference/connection-string/#dns-seed-list-connection-format>` instead.
   This format offers more flexibility of deployment and lets you change the
   servers in rotation without reconfiguring clients.
```

this part of the connection URI.

The next part of the connection string specifies the hostname or IP address and

> port of your MongoDB instance. In the example above, we use ``sample.host``

Reviewer: 👍

.. tip::

>    You only need to specify one host to connect to a replica set.
>    However, to ensure connectivity if the specified host is unavailable,
>    provide the full list of hosts.

Reviewer 2: S:

```suggestion
   Provide the full list of hosts when connecting to a replica set. 
   This ensures connectivity even if one of the hosts is unavailable. 
```
